ansible/roles/compute_init/README.md

```diff
@@ -42,10 +42,35 @@ The following roles/groups are currently fully functional:
   node and all compute nodes.
 - `openhpc`: all functionality
 
-# Development/debugging
+All of the above are defined in the skeleton cookiecutter config, and are
```

Comment:
I think this is a bit misleading. I'd replace this entire para with something like:

> The above may be enabled by setting the `compute_init_enable` property on the terraform `compute` variable